File: 02 Digits and Numbers - Foundation Programs/program-09.cpp
// Write a program that takes a 4-digit number and calculates the average of its first and last digits.

#include<iostream>
using namespace std;
int main()
{
    int number,first_digit,last_digit,avg;
    cout<<"Enter 4-digit number : ";
    cin>>number;
    
    last_digit=number%10;  
    first_digit=number/1000;
    avg=(first_digit+last_digit)/2;

    cout<<"First Digit : "<<first_digit<<endl<<"last digit : "<<last_digit<<endl;
    cout<<"The average of its first and last digits : "<<avg;

    return 0;
}

File: archit_vishwakarma/Level-1/03-Digits-And-Numbers/program-09.py
n = input("\nEnter a number : ")

try:
    n = int(n)
    if n < 10000 and n > 999:

        a = n//1000
        b = n%10
        
        print(f"Average of first and last digit : {(a+b)/2}\n")

    else:
        print("Enter a FOUR Digit Number")
except ValueError:
    print("enter a valid number")


